test/hashesTest.ts

```diff
@@ -149,4 +164,44 @@ describe('Ledger', function () {
       assert.equal(actualEntryHash, expectedEntryHash)
     })
   })
+
+  describe('Transaction Hash Verification', function () {
```

Comment:
why is this not part of the above `describe`?

Reply:
The rest of the tests in this file have a structure of `Describe( functionName )` followed by `it( 'will do this thing and see what happens' ) - @mvadari, Should I rewrite the structure to match that of the other tests we have?